instrumentation/opentelemetry-instrumentation-grpc/setup.cfg

```diff
@@ -55,4 +55,3 @@ where = src
 [options.entry_points]
 opentelemetry_instrumentor =
     grpc_client = opentelemetry.instrumentation.grpc:GrpcInstrumentorClient
-    grpc_server = opentelemetry.instrumentation.grpc:GrpcInstrumentorServer
```

Comment:
Why remove this? I'm not super familiar but I think this is for autoinstrumentation

Reply:
Oversight - I started going a different way and then realized this needed to stay. I'll add it back.